.kitchen.yml

```diff
@@ -99,3 +101,9 @@ platforms:
 
 suites:
   - name: default
+
+verifier:
+  name: inspec
+  inspec_tests:
+    - path: test/integration/1-system
```

Comment:
That part is for Vagrant stuff we're working on in `packer-st2` repo.
Better to remove that and leave `stackstorm` tests only.